script/functions.js

```diff
@@ -67,5 +67,7 @@ function formatString(format) {
  * Custom onSelect handler for struct img button
  */
 window.insertStructMedia = function (edid, mediaid, opts, align) {
+    // escape dots in field id, bureaucracy plugin uses dots and it can break jQuery selectors
+    edid = edid.split('.').join('\\.');
```

Comment:
how about using document.getElementById(edid) instead of building a selector string below?

Reply:
Thank you, I was blinded by jQuery again.